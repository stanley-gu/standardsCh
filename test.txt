% My test pandoc-ument

I want to reference this: \cite{Gepasi1993}

# References