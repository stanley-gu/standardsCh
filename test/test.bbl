% $ biblatex auxiliary file $
% $ biblatex version 1.7 $
% Do not modify the above lines!
%
% This is an auxiliary file used by the 'biblatex' package.
% This file may safely be deleted. It will be recreated as
% required.
%
\begingroup
\makeatletter
\@ifundefined{ver@biblatex.sty}
  {\@latex@error
     {Missing 'biblatex' package}
     {The bibliography requires the 'biblatex' package.}
      \aftergroup\endinput}
  {}
\endgroup

\entry{Gepasi1993}{article}{}
  \name{author}{1}{}{%
    {{}%
     {Mendes}{M.}%
     {P.}{P.}%
     {}{}%
     {}{}}%
  }
  \strng{namehash}{MP1}
  \strng{fullhash}{MP1}
  \field{sortinit}{M}
  \field{pages}{563\bibrangedash 571}
  \field{title}{GEPASI: A software package for modelling the dynamics, steady
  states and control of biochemical and other systems.}
  \field{volume}{9}
  \field{journaltitle}{Comput. Applic. Biosci.}
  \field{year}{1993}
\endentry

\lossort
\endlossort

\endinput
