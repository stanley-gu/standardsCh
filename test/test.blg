This is BibTeX, Version 0.99d (TeX Live 2012)
Capacity: max_strings=35307, hash_size=35307, hash_prime=30011
The top-level auxiliary file: test.aux
The style file: biblatex.bst
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated wiz_functions (elt_size=4) to 6000 items from 3000.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Database file #1: test-blx.bib
Database file #2: test.bib
Biblatex version: 1.7
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated wiz_functions (elt_size=4) to 9000 items from 6000.
You've used 2 entries,
            6019 wiz_defined-function locations,
            1128 strings with 8188 characters,
and the built_in function-call counts, 3899 in all, are:
= -- 156
> -- 136
< -- 25
+ -- 56
- -- 48
* -- 197
:= -- 441
add.period$ -- 0
call.type$ -- 2
change.case$ -- 10
chr.to.int$ -- 34
cite$ -- 2
duplicate$ -- 317
empty$ -- 395
format.name$ -- 48
if$ -- 885
int.to.chr$ -- 0
int.to.str$ -- 2
missing$ -- 0
newline$ -- 40
num.names$ -- 41
pop$ -- 353
preamble$ -- 1
purify$ -- 11
quote$ -- 0
skip$ -- 233
stack$ -- 0
substring$ -- 154
swap$ -- 101
text.length$ -- 40
text.prefix$ -- 1
top$ -- 1
type$ -- 64
warning$ -- 0
while$ -- 68
width$ -- 0
write$ -- 37
