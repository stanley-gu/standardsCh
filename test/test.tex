\documentclass[]{article}
\usepackage[T1]{fontenc}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
% use microtype if available
\IfFileExists{microtype.sty}{\usepackage{microtype}}{}
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \usepackage{fontspec}
  \ifxetex
    \usepackage{xltxtra,xunicode}
  \fi
  \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
  \newcommand{\euro}{â‚¬}
\fi
\usepackage{biblatex}
\bibliography{test}
\ifxetex
  \usepackage[setpagesize=false, % page size defined by xetex
              unicode=false, % unicode breaks when used with xetex
              xetex]{hyperref}
\else
  \usepackage[unicode=true]{hyperref}
\fi
\hypersetup{breaklinks=true,
            bookmarks=true,
            pdfauthor={Stanley Gu},
            pdftitle={My test pandoc-ument},
            colorlinks=true,
            urlcolor=blue,
            linkcolor=magenta,
            pdfborder={0 0 0}}
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\setcounter{secnumdepth}{0}

\title{My test pandoc-ument}
\author{Stanley Gu}
\date{}

\begin{document}
\maketitle

\section{Top}

I want to reference this: \cite{Gepasi1993}

\printbibliography

\end{document}
