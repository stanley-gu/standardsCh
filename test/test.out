\BOOKMARK [1][-]{section*.1}{\376\377\000T\000o\000p}{}% 1
